Libraries/LibWeb/DOM/AbortSignal.cpp

```diff
@@ -35,14 +35,24 @@ void AbortSignal::initialize(JS::Realm& realm)
 }
 
 // https://dom.spec.whatwg.org/#abortsignal-add
-void AbortSignal::add_abort_algorithm(Function<void()> abort_algorithm)
+Optional<AbortSignal::AbortAlgorithmID> AbortSignal::add_abort_algorithm(Function<void()> abort_algorithm)
 {
+    static AbortAlgorithmID s_next_id = 0;
```

Comment:
should this ID allocator *really* be static? It's only used to allocate IDs for a member variable on each abort signal instance that are not shared, no?

Reply:
tru, made it a member variable